initialize/init.go

```diff
@@ -19,7 +22,20 @@ import (
 
 const timeoutDuration = 30 * time.Second
 
-func InitializeOnce(ctx context.Context, initOnceCompletedPath, passwordFilePath, miscConfPath string) error {
+// MyConfTemplateParameters define parameters for a MySQL configuration template.
+type MyConfTemplateParameters struct {
+	// ServerID is the value for server_id of MySQL configuration
+	ServerID uint32
+	// AdminAddress is the value for admin_address of MySQL configuration
+	AdminAddress string
+}
+
+func InitializeOnce(ctx context.Context, initOnceCompletedPath, passwordFilePath, miscConfPath string, serverIDBase uint32) error {
+	log.Info("generate mysql configuration file", nil)
+	if err := generateMySQLConfiguration(ctx, serverIDBase, moco.MySQLConfTemplatePath, moco.MySQLConfPath, moco.MySQLConfName); err != nil {
+		return err
+	}
+
 	_, err := os.Stat(initOnceCompletedPath)
 	if err == nil {
```

Comment:
Please use the `err` variable as in the other parts.

```suggestion
	err := generateMySQLConfiguration(ctx, serverIDBase, moco.MySQLConfTemplatePath, moco.MySQLConfPath, moco.MySQLConfName)
	if err != nil {
		return err
	}

	_, err = os.Stat(initOnceCompletedPath)
	if err == nil {
```